coil-svg/src/main/java/coil/decode/SvgDecoder.kt

```diff
@@ -3,24 +3,32 @@
 package coil.decode
 
 import android.content.Context
-import android.graphics.Bitmap
 import android.graphics.Canvas
-import android.os.Build.VERSION.SDK_INT
+import android.graphics.RectF
+import android.graphics.drawable.Drawable
 import androidx.core.graphics.drawable.toDrawable
 import coil.bitmap.BitmapPool
 import coil.size.OriginalSize
 import coil.size.PixelSize
 import coil.size.Size
 import coil.util.indexOf
+import coil.util.toSoftware
 import com.caverock.androidsvg.SVG
 import okio.BufferedSource
 import okio.ByteString.Companion.encodeUtf8
 import okio.buffer
 
 /**
  * A [Decoder] that uses [AndroidSVG](https://bigbadaboom.github.io/androidsvg/) to decode SVG files.
+ *
+ * @param context A [Context] used to create the [Drawable].
+ * @param useViewBoundsAsIntrinsicSize If true, uses the SVG's view bounds as the intrinsic size for the SVG.
+ *  If false, uses the SVG's width/height as the intrinsic size for the SVG.
  */
-class SvgDecoder(private val context: Context) : Decoder {
+class SvgDecoder @JvmOverloads constructor(
+    private val context: Context,
+    private val useViewBoundsAsIntrinsicSize: Boolean = true
```

Comment:
Is it okay that this changes behavior for those who are using this currently? Maybe it should default to false until a 1.2 or a 2.0 so that folks don't update their library and end up with unexpected behavior? Otherwise, I'd just make sure its communicated clearly in the CHANGELOG.

Reply:
@Jawnnypoo Good point. Will call this out at the top of the release notes.

I'm definitely for only enabling this in `1.2` (which will be the next release). `2.0` is still a ways away so I'd want to introduce the behaviour change before then. Also I'm hoping this change will be invisible for most users as for most SVGs `width`/`height` already has the same aspect ratio as `viewBox`. This change mostly improves the handling for weird SVGs that Coil wouldn't render properly.

Reply:
Gotcha! Thanks for clarifying 👍